gpflow/models/gplvm.py

```diff
@@ -78,13 +81,13 @@ def __init__(self, X_mean, X_var, Y, kern, M, Z=None, X_prior_mean=None, X_prior
         assert X_var.shape[0] == Y.shape[0], 'X var and Y must be same size.'
 
         # inducing points
-        if Z is None:
+        if Z is None and feat is None:
             # By default we initialize by subset of initial latent points
             Z = np.random.permutation(X_mean.copy())[:M]
-        else:
-            assert Z.shape[0] == M
-        self.Z = Parameter(Z)
-        self.num_latent = Z.shape[1]
+
+        self.feature = features.inducingpoint_wrapper(feat, Z)
+        assert self.feature.Z.shape[0] == M
```

Comment:
This implies that even though you've moved to features, this class will still only support `InducingPoints`. If this model cannot support other features, then might as well keep it simple and just stick with `Z`. If it _can_ now support any feature, I think it'd be good to clean up the `__init__` signature and and remove the `M` argument entirely (and then don't bother with the inducingpoint_wrapper).